components/nimbus/old-nimbus-sdk-toplevel-README.md

```diff
@@ -0,0 +1,42 @@
+# nimbus-sdk [![CircleCI](https://circleci.com/gh/mozilla/nimbus-sdk/tree/main.svg?style=svg)](https://circleci.com/gh/mozilla/nimbus-sdk/tree/main)
+Cross Platform Rapid Experiments "Nimbus" SDK
```

Comment:
With the "cutting a release" stuff removed, I think I would be happy for this file to remain as `README.md` and serve as a component-level readme, and we can iterate on it over time to improve consistency with other components. WDYT?

Reply:
Done.